src/components/bookCard.jsx

```diff
@@ -1,49 +1,61 @@
 import PropTypes from 'prop-types';
 import './styles/bookcard.css';
+import { useDispatch } from 'react-redux';
 import CircularProgress from './Progress';
+import { removeBook } from '../redux/books/booksSlice';
 
 const BookCard = ({
-  category, title, author, percentValue, chapter,
-}) => (
-  <>
-    <div className="card-div">
-      <article className="card">
-        <div className="content-title">
-          <p className="category">{category}</p>
-          <h2 className="title">{title}</h2>
-          <small className="author">{author}</small>
-        </div>
-        <div className="actions">
-          <button type="button" className="action">Comments</button>
-          <span className="actions-line" />
-          <button type="button" className="action">Remove</button>
-          <span className="actions-line" />
-          <button type="button" className="action">Edit</button>
-        </div>
-      </article>
-      <div className="current-chapter">
-        <div className="progress">
-          <CircularProgress />
-          <div className="percent">
-            <p className="value">
-              {percentValue}
-              %
-            </p>
-            <p className="completed">Completed</p>
+  id, category, title, author, percentValue, chapter,
+}) => {
+  // const data = useSelector((state) => state.book);
```

Comment:
- [ ] Kindly remove all the commented code, this will make your code clean and readable.